Description: airodump-ng can't find the IEE OUI file.
 We are doing here the minium changes to keep release team
 happy and have this allowed into Jessie.
 .
 Next upstream release of aircrack-ng will already contain
 r2455, so this patch can be dropped.
Author: Carlos Alberto Lopez Perez <clopez@igalia.com>
Bug-Debian: http://bugs.debian.org/771221
Applied-Upstream: http://trac.aircrack-ng.org/changeset/2455
--- a/src/airodump-ng.h
+++ b/src/airodump-ng.h
@@ -146,8 +146,8 @@
 
 #define OUI_PATH0 "/etc/aircrack-ng/airodump-ng-oui.txt"
 #define OUI_PATH1 "/usr/local/etc/aircrack-ng/airodump-ng-oui.txt"
-#define OUI_PATH2 "/usr/share/aircrack-ng/airodump-ng-oui.txt"
-#define OUI_PATH3 "/usr/share/misc/oui.txt"
+#define OUI_PATH2 "/var/lib/ieee-data/oui.txt"
+#define OUI_PATH3 "/usr/share/ieee-data/oui.txt"
 #define MIN_RAM_SIZE_LOAD_OUI_RAM 32768
 
 int read_pkts=0;
